app/src/main/java/com/sdp/movemeet/models/User.java

```diff
@@ -111,11 +118,25 @@ public String getDocumentPath() {
         return this.documentPath;
     }
 
+    /**
+     * @return the list of activities created by the user
+     */
+    public ArrayList<String> getCreateActivitiesId() {
```

Comment:
Here as well, I would rather name this getter "getCreatedActivitiesId" (with a "d").